### PR TITLE
Add logging to ztls when acting as the server

### PR DESCRIPTION
**Pull request by aaspring**
Branch name is inaccurate b/c scope changed. ClientHello is now logged, but everything else is too.

### CODE REVIEW

File: ztools/ztls/common.go [hunk 1]
```diff
@@ -102,12 +104,81 @@ const (
...
+	}
+	if expectedName := nameForCurve(aux.Value); expectedName != aux.Name {
+		return fmt.Errorf("mismatched curve and name, curve: %d, name: %s, expected name: %s", aux.Value, aux.Name, expectedName)
+	}
+	*curveID = CurveID(aux.Value)
+	return nil
+}
+
+type PointFormat uint8
+
```
Comment (dadrian) [inline ztools/ztls/common.go]: Did you make this change? Didn't Zakir implement this a while ago? Is something up with this merge? Are you rebased onto the most recent master?

Comment (aaspring) [inline ztools/ztls/common.go]: I added this. I did not see the logging anywhere. @zakird??

Comment (dadrian) [inline ztools/ztls/common.go]: Ok, I guess I'm just crazy.

**Comment by zakird** [comment]
This looks fine to me.

File: ztools/ztls/common.go (hunk 1 above)
Comment (zakird) [inline ztools/ztls/common.go]: I don't remember ever adding something like this.

**Pull request merged**